husky_control/CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog for package husky_control
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.6.5 (2022-09-26)
+------------------
+* Add keyboard teleoperation functionality.
+* Contributors: Tinker Twins (Chinmay Samak & Tanmay Samak)
+
```

Comment:
Please do not edit the changelog directly; this gets automatically updated when the package is released

Reply:
Sorry, did not know that this gets updated automatically when the package is released. The requested changes have been made.